vms/platformvm/state/subnet_only_validator.go

```diff
@@ -0,0 +1,92 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package state
+
+import (
+	"fmt"
+
+	"github.com/google/btree"
+
+	"github.com/ava-labs/avalanchego/database"
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/vms/platformvm/block"
+)
+
+var _ btree.LessFunc[*SubnetOnlyValidator] = (*SubnetOnlyValidator).Less
+
+type SubnetOnlyValidator struct {
```

Comment:
Should we also add StartTime as a field here?

Reply:
Yeah we'll need that for the WARP message + uptimes 👍